act-rules-format.bs

```diff
@@ -110,42 +110,90 @@ An ACT Rule MUST have a description that is in plain language, and provides a br
 
 <div class=example>
-  <p><strong>Example</strong>: The outcome of this rule maps to the accessibility requirements in the following manner:</p>
-  <ul>
-    <li>`failed`: The success criteria are *not satisfied*</li>
-    <li>`passed` or `inapplicable`: Further testing is necessary to determine if the success criteria are satisfied</li>
-  </ul>
+  <p><strong>Example:</strong>Accessibility Requirements Mapping for a rule that tests that each `img` element has an `alt` attribute</p>
+  <blockquote><ul>
+    <li><a href="https://www.w3.org/TR/WCAG20-TECHS/H37.html">
+      Technique H37: Using alt attributes on img elements
+      </a><ul>
+        <li><strong>Not required</strong> for conformance to WCAG 2.0 and WCAG 2.1 at any level</li>
+        <li>Outcome mapping: <ul>
+          <li>Failed outcome: not satisfied</li>
+          <li>Passed outcome: satisfied</li>
+          <li>Inapplicable outcome: satisfied</li>
+        </ul></li>
+      </ul>
+    </li>
+    <li><a href="https://disic.github.io/rgaa_referentiel_en/criteria.html#test-1-1-1">
+      RGAA 3, Test 1.1.1: Does each image have a text alternative
+      </a><ul>
+        <li><strong>Required for conformance</strong> to RGAA 3 level A</li>
+        <li>Outcome mapping: <ul>
+          <li>Failed outcome: not satisfied</li>
+          <li>Passed outcome: satisfied</li>
+          <li>Inapplicable outcome: satisfied</li>
+        </ul></li>
+      </ul>
+    </li>
+  </ul></blockquote>
 </div>
 
-<div class=note>
-  **Note**: In WCAG 2.1, success criteria do not evaluate to `passed`, `failed` or `inapplicable`. Rather they can be *satisfied* (or not). (See the [WCAG 2.1 definition: satisfies a success criterion](https://www.w3.org/TR/WCAG21/#dfn-satisfies).) If a success criterion is *not satisfied* a web page can only conform if there is a conforming alternative version, as described in [WCAG 2.1 Conformance Requirement 1](https://www.w3.org/TR/WCAG21/#cc1).
+If the `failed` outcome can not be mapped to *not satisfied* for an accessibility requirement, that requirement MUST NOT be included in the accessibility requirements mapping. The optional Background section could be used to list accessibility requirements and standards that may be thematically related, but for which the rule is not a failure test.
+
+If the rule does not map to any [=accessibility requirement=], the accessibility requirement mapping will only contain the explainer that it is not required for conformance to the [=accessibility requirements document=]. This is common with atomic rules used in composite rules, where accessibility requirements are only not satisfied with multiple atomic rules have `failed`.
```

Comment:
"with multiple atomic rules have `failed`" --> "when multiple atomic rules have the outcome `failed` "